bmwqemu.pm

```diff
@@ -126,7 +126,7 @@ sub init {
 
     unless ($vars{CASEDIR}) {
         die "DISTRI undefined\n" . pp(\%vars) . "\n" unless $vars{DISTRI};
-        my @dirs = ("$scriptdir/distri/$vars{DISTRI}");
+        my @dirs = ("$scriptdir/tests/$vars{DISTRI}");
```

Comment:
I don't get the meaning of "tests/" dir. Is this also only a nice "default here?